src/modules/rabbitmq/blockchain-events/handlers/updatePrice-event.handler.ts

```diff
@@ -13,7 +13,7 @@ import { MarketplaceTypeEnum } from 'src/modules/marketplaces/models/Marketplace
 import { UsdPriceService } from 'src/modules/usdPrice/usd-price.service';
 import { BigNumberUtils } from 'src/utils/bigNumber-utils';
 import { DEADRARE_KEY } from 'src/utils/constants';
```

Comment:
here you need to revert the changes. update price event handler should remain the same

Reply:
Done